conjure-java-jaxrs-client/src/test/java/com/palantir/conjure/java/client/jaxrs/feignimpl/HeaderAccessUtilsTest.java

```diff
@@ -25,8 +25,9 @@
 
 public final class HeaderAccessUtilsTest {
     private static final ImmutableMap<String, Collection<String>> TEST_HEADERS_MAP = ImmutableMap.of(
-            "header", Arrays.asList("value1"), "Header", Arrays.asList("value2", "value3"), "HEADER", Arrays.asList(
-                    "value4", "value5"));
+            "header", Arrays.asList("value1"),
+            "Header", Arrays.asList("value2", "value3"),
+            "HEADER", Arrays.asList("value4", "value5"));
```

Comment:
@dansanduleac did we drop back into some special-casing for ImmutableMap??

Reply:
so, the previous commit wrecked the original formatting of this, and the logic that keeps pairs as they are doesn't work unless they are already paired before formatting.